src/main/kotlin/com/hibob/project/dao/feedbackConfigurationDao.kt

```diff
@@ -0,0 +1,50 @@
+package com.hibob.project.dao
+
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class FeedbackDao @Autowired constructor(private val sql: DSLContext) {
+
+    private val feedbackTable = FeedbackConfigurationTable.instance
+
+    private val feedbackConfigurationMapper = RecordMapper<Record, FeedbackConfiguration>
```

Comment:
the standard in kotlin is this:
private val feedbackConfigurationMapper = RecordMapper<Record, FeedbackConfiguration> { record ->
...
}